### ISSUE TITLE
$USERPROFILE/.ssh/id_rsa

### ISSUE THREAD

**Issue by dzmitry-lahoda** [SchoofsKelvin/vscode-sshfs#151]
That

```
A path to a private key. Supports environment variables, e.g. `$HOMEDRIVE$HOMEPATH/.ssh/myKey.ppk` or `$HOME/.ssh/myKey`
```
into 
```
A path to a private key. Supports environment variables, e.g. `$HOMEDRIVE$HOMEPATH/.ssh/myKey.ppk` or `$HOME/.ssh/myKey` or `$USERPROFILE/.ssh/id_rsa`
```
which shows Windows works. 

I would even prefer that `$USERPROFILE/.ssh/id_rsa` could be default value of config.

**Comment by SchoofsKelvin** [comment]
The `$HOMEDRIVE$HOMEPATH/.ssh/myKey.ppk` is already meant for Windows, as `$HOMEDRIVE` resolves to e.g. `C:\` while `$HOMEPATH` resolves to e.g. `Users\username`. I didn't know about the `$USERPROFILE` variable, so I will be changing the suggestion from the former to the latter.

**Comment by SchoofsKelvin** [comment]
I've changed the suggestion (ba79853) and published it under version 1.16.3 of the extension.
